doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# WELCOME TO YOUR 
+# CONTRIBUTION GUIDELINES
+
+
+To help you save on time and hassle down the road, 
+these contribution guidelines will help for submitting well-formed pull requests and opening useful issues.
+
+For the repository owner, and contributors these guidelines are a way to communicate how people should contribute.
+
+The following is laid out to help govern and champion a productive collabrative effort.
+
+When deciding a title/name for your pull request please prefix with:
+a. (Feat): This is a new feature
+b. (Fix): I am fixing something
+c. (Doc): Some new documentation
```

Comment:
The prefixes are only 
- `(Feat):`
- `(Fix):`
- `(Doc):`
and then the title follows. 

I've read this as I need to put `(Feat): This is a new feature and then the title I want for the PR`. E.g. if I was creating a new Navbar the title of my PR would be `(Feat): This is a new feature and the new feature is a Navbar`, but the title should be something like `(Feat): Add a Navbar component`

Reply:
I slightly reworded this paragraph in the document